components/core/tools/scripts/lib_install/macos/install-all.sh

```diff
@@ -16,6 +16,7 @@ brew install \
   go-task \
   java11 \
   libarchive \
+  llvm@16 \
```

Comment:
What happens if the user already has, for instance, llvm@19 installed?

Reply:
As discussed offline, brew installs different versions of llvm clang toolchains side-by-side, so this should be a non-issue.